lib/datastack/server/ds_class.py

```diff
@@ -262,7 +264,7 @@ def html(self, html, id=''):
         }
         self.append_block(block)
 
-    def editable_html(self, key):
+    def editable_html(self, key, id=''):
         default_html = [
       {
         "_id": "5f54d75b114c6d176d7e9765",
```

Comment:
is this required?

Reply:
It is standards that we put id to all widgets including editable html